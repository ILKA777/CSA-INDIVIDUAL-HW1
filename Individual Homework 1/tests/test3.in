12
10 19 -3 -4 5 -1234 -11 444 -3 322 -1000 -24
