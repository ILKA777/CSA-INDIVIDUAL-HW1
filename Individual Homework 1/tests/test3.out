800-2279800-2279800-2279800-2279800-2279800-2279
